<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>diagSelectExamples</class>
 <widget class="QDialog" name="diagSelectExamples">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Nihongo Companion</string>
  </property>
  <widget class="QProgressBar" name="pbSearch">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tExamples">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>581</width>
     <height>311</height>
    </rect>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Japanese</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>English</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="cbField_out">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelField_out">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Field to append to:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bConfirm">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bCancel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
